backend/requirements.txt

```diff
@@ -1,28 +1,50 @@
 asgiref==3.4.1
+attrs==21.4.0
```

Comment:
Clean up these requirements. We only need to install jsmin==3.0.0

Reply:
Can we also remove:
```
asgiref==3.4.1
backports.zoneinfo==0.2.1
pytz==2021.3
sqlparse==0.4.2
mccabe==0.6.1
six==1.16.0
```

Reply:
These are support libraries of Django, It will be installed with it anyway

Reply:
Next time when we install a dependencies, the command should be:
```bash
pip install <the-library>
pip freeze | grep <the-library> >> requirements.txt
```